### PR TITLE
Dashboards contracts assignee page

### LINKED ISSUE #3091: Contracts / Asignee page
Let's create a page for contract assignees, and link them to this page

For the moment include:

- name
- id
- list of contracts

Things to decide:

- identifier in the URL. Can we use the assignee ID (sometimes missing), or should we use a hash?

### LINKED ISSUE #3112: Dashboards / Change JS folders structure

### PR DESCRIPTION
**Pull request by eltercero**
Closes #3091
Closes #3112

## :v: What does this PR do?

Adds an assignee page. It's accesible from each of the assignees table rows in the Summary page and from a link in the assignee name that appears in any contract show view.

## :mag: How should this be manually tested?

It's in staging. While in the page, it should display the contracts the assignee has. Filters from the sidebar should work too.

## :eyes: Screenshots

Captura de pantalla 2020-05-27 a las 10 55 38

## :shipit: Does this PR changes any configuration file?

No

## :book: Does this PR require updating the documentation?

No

### CODE REVIEW

**Comment by eltercero** [comment]
@ferblape @Crashillo ready to review!

File: app/javascript/gobierto_dashboards/webapp/components/TableRow.vue [hunk 1]
```diff
@@ -36,10 +36,15 @@ export default {
...
       default: ''
+    },
+    routingAttribute: {
+      type: String,
+      default: 'id'
     }
   },
   created(){
     this.initFormattedItem();
+    this.routingId = this.item[this.routingAttribute];
```
Comment (Crashillo) [inline app/javascript/gobierto_dashboards/webapp/components/TableRow.vue:48]: Initialize in data

Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/components/TableRow.vue:48]: Please, install `eslint-plugin-vue`

imagen

File: app/javascript/gobierto_dashboards/webapp/containers/contracts/AssigneesShow.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  methods: {
+    buildItems(){
+      const assigneeRoutingId = this.$route.params.id;
+      const contracts = this.contractsData.filter(({ assignee_routing_id }) => assignee_routing_id === assigneeRoutingId ) || [];
+
+      this.items = contracts
+      this.columns = assigneesShowColumns;
+
+      if (contracts.length > 0) {
+        const contract = contracts[0]
```
Comment (Crashillo) [inline app/javascript/gobierto_dashboards/webapp/containers/contracts/AssigneesShow.vue:69]: 👀

**Review by Crashillo — CHANGES_REQUESTED** [review]
It's strident that Contracts, Tenders, Subsidies.... apparently are all the same, or very similar, and everything is triplicated

**Review by Crashillo — APPROVED** [review]

File: app/javascript/gobierto_dashboards/webapp/containers/contracts/AssigneesShow.vue (hunk 2 above)
Comment (eltercero) [reply app/javascript/gobierto_dashboards/webapp/containers/contracts/AssigneesShow.vue:69]: I know, I know! But I just need one of the contracts to get the common assignee information.

File: app/javascript/gobierto_dashboards/webapp/components/TableRow.vue (hunk 1 above)
Comment (eltercero) [reply app/javascript/gobierto_dashboards/webapp/components/TableRow.vue:48]: Installed. Yeah, there were a lot of warning, fixed most of them.

**Pull request merged**